packages/rfw/test/material_widgets_test.dart

```diff
@@ -23,6 +25,9 @@ void main() {
   }
 
   testWidgets('Material widgets', (WidgetTester tester) async {
+    setUpTolerantComparator(
+        testPath: 'test/material_widget_test.dart', precisionTolerance: 0.01);
```

Comment:
how much is 0.01?

Reply:
`0.01` is `4800px` out of the `480000px` in the `material_test.scaffold.png` file. 

The actual diff from the failing test is `4px` so a more precise value for the tolerance is `0.00000833333`.

Reply:
Meaning there are four pixels that have a non-identical match to the control?

Reply:
I worry that allowing four pixels to be just flat out wrong means that if we start incorrectly rendering a dot somewhere, we won't notice it. "Tolerance" makes it sound like we'd accept it if the four pixels happened to be slightly off in color (e.g. if 0xFFFEFEFE became 0xFFAFAFA or something), not that we'd accept wildly wrong renderings (e.g. 0xFF00FF00 becoming 0xFFFF0000).

Reply:
True. What would be a better fix for the issue? Updating the control images? 

Also, I implemented the tolerance based on this comment in the original issue: https://github.com/flutter/flutter/issues/151611#issuecomment-2223874000.

Reply:
Updating the control images is a short-term fix, it doesn't really address the underlying issue (which is that skia and impeller don't guarantee pixel-identical rendering across versions for things like antialiasing and blurs, but the test is assuming they do).

I don't have a good solution. Setting the tolerance to something like 0.00002 (so if I understood correctly, allowing about 10 pixels to drift from the control) might be ok I guess? In the absence of any better ideas...

Reply:
Yes, 10px away from the control. 

I'll update the tolerance.

Reply:
Updated now in 0fd9943.